lib/components/src/blocks/PropsTable/PropRow.tsx

```diff
@@ -18,21 +21,34 @@ const Required = styled.span(({ theme }) => ({
   cursor: 'help',
 }));
 
-const Description = styled.div({
+const Description = styled.div(({ theme }) => ({
   '&&': {
     p: {
       margin: '0',
     },
   },
-});
+  code: codeCommon({ theme }),
+  '& code': {
+    lineHeight: '18px',
+    margin: 0,
+    display: 'inline-block',
+  },
+}));
 
 const Type = styled.div<{ hasDescription: boolean }>(({ theme, hasDescription }) => ({
-  color: theme.color.dark,
-  marginTop: hasDescription ? '3px' : '0',
+  color:
+    theme.base === 'light'
+      ? transparentize(0.1, theme.color.defaultText)
+      : transparentize(0.2, theme.color.defaultText),
+  marginTop: hasDescription ? '4px' : '0',
 }));
 
 const TypeWithJsDoc = styled.div(({ theme }) => ({
-  color: theme.color.darker,
+  color:
+    theme.base === 'light'
+      ? transparentize(0.1, theme.color.defaultText)
+      : transparentize(0.2, theme.color.defaultText),
+  fontSize: theme.typography.size.s2 - 1,
```

Comment:
Since you already defined this fontSize in PropValue for Text and Expandable, I dont think you need to define it here, im I wrong?

Reply:
nice will fix